tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,8 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
The feedback from the Wave 1 tests applies here as well. A couple points to consider for testing: 
- asserting that specific items are not in a collection doesn't guarantee that the collection is empty, it only ensures that those items are not in it. 
- checking `item not in vendor.inventory` doesn't guarantee that the list is unchanged, just that `item` isn't present in the list. 

If we want to confirm that there weren't side effects or changes to either vendor, what could we assert about each vendor's inventory to ensure that their contents did not change?